tools/@o3r/test-helpers/project.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "test-helpers",
+  "$schema": "https://raw.githubusercontent.com/nrwl/nx/master/packages/nx/schemas/project-schema.json",
+  "projectType": "library",
+  "sourceRoot": "tools/@o3r/test-helpers/src",
+  "prefix": "o3r",
+  "targets": {
+    "build": {
+      "executor": "nx:noop",
+      "dependsOn": [
+        "build-builders",
+        "compile"
+      ]
+    },
+    "compile": {
```

Comment:
As it need a compilation, should not it make more sense to be in `packages` folder

Reply:
moved